lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -1000,6 +845,104 @@ sub _resolve_workflow_for_build {
                 );
             }
         }
+        #Add gene category annotations to some output files from steps above. (e.g. determine which SNV affected genes are kinases, ion channels, etc.)
+        #AnnotateGenesByCategory - gene_category_<exome|wgs|wgs_exome>_<snv|indel>_result
+        my $annotate_genes_by_category_op = $self->annotate_genes_by_category_op($workflow);
+        $workflow->create_link(
+            source               => $converge_snv_indel_report_ops[-1],
```

Comment:
It's our naive attempt at picking the snv indel report that was processed with the highest minimum mapping qualities. sireport_min_mq_bq is supposed to be a ''Comma separated increasing-list of minimum mapping qualities for bam-readcounting." Although I don't believe that that is enforced in any way. I wonder if the parse_qualities should somehow attempt to sort the quality cutoffs. @gatoravi @jasonwalker80 what do you think?